x/ibc/07-tendermint/types/client_state.go

```
@@ -67,6 +76,19 @@ func (cs ClientState) GetFrozenHeight() uint64 {
 	return cs.FrozenHeight
 }
 
+//Unfreeze unfreezes light client after misbehaviour and clears any frozen height previously set
+func (cs *ClientState) Unfreeze() error {
```

Actually I don't think this function should return an error